packages/webapp/webapp_server.js

```
@@ -1424,10 +1423,15 @@ async function runWebAppServer() {
 
     if (unixSocketPath) {
-      if (cluster.isWorker) {
-        const workerName = cluster.worker.process.env.name || cluster.worker.id;
-        unixSocketPath += '.' + workerName + '.sock';
-      }
+      // Lazy-load cluster only when needed (UNIX_SOCKET_PATH with workers).
+      // Avoids loading the module in the common case where it is unused.
+      try {
+        const cluster = require('cluster');
+        if (cluster.isWorker) {
+          const workerName = cluster.worker.process.env.name || cluster.worker.id;
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)